sdk/schemaregistry/Azure.Data.SchemaRegistry/api/Azure.Data.SchemaRegistry.netstandard2.0.cs

```diff
@@ -7,6 +7,8 @@ namespace Azure.Data.SchemaRegistry
         private readonly int _dummyPrimitive;
```

Comment:
Not sure if you meant to, but there is an empty sc.txt file.

Reply:
That got checked in accidentally!